meta-lmp-base/recipes-support/resize-helper/resize-helper_0.1.bb

```diff
@@ -5,7 +5,7 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-2-Clause;md5=cb641bc04cda31
 
 inherit allarch systemd
 
-RDEPENDS:${PN} += "e2fsprogs-resize2fs gptfdisk parted util-linux-findmnt"
+RDEPENDS_${PN} += "e2fsprogs-resize2fs gptfdisk util-linux-findmnt"
```

Comment:
For main, it's `RDEPENDS:${PN}`

Reply:
@Tim-Anderson syntax will need to be updated.

Reply:
corrected